rfc-030-auditing.md

```diff
@@ -0,0 +1,41 @@
+# Introduce auditing on all Rails projects
+
+## Summary
+
+We should routinely use auditing in our projects. For those projects which will
+undergo a Service Assessment, this is a requirement; for others it is good 
+practice to ensure confidence and security in the data we collect & store.
+
+## Problem
+
+For all projects undergoing a [GOV.UK Service Assessment](https://www.gov.uk/service-manual/service-assessments), 
```

Comment:
This is great context to include 👍

Reply:
Another reason we added it on BEIS was to act as a risk mitigation. _If_ a user made neglient or malicious changes we could more quickly identify the compromised user/s and the extent of the damage made to our data, allowing us to take corrective actions without having to do a full audit.